scripts/partition_manager/partition_manager.rst

```diff
@@ -536,11 +536,28 @@ See :ref:`ug_bootloader_external_flash` for more details on using external flash
 
 .. _pm_build_system:
 
-You must enable device drivers for each flash partition.
-Set the ``DEFAULT_DRIVER_KCONFIG`` parameter to the Kconfig option that enables the driver during compilation.
-When a driver for a partition is not enabled, the partition will not be included in the partition manager and will not be accessible at runtime.
-You can use the :kconfig:option:`CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK` Kconfig option to override the behavior and prevent compile-time check for external flash memory devices.
-This is useful when the external device does not have a driver controlled by Kconfig options.
+For a partition to be accessible at run-time, a flash device it is placed on needs to have its driver enabled at compile time.
+Partition manager will ignore partitions that are located in a region without a driver associated with it.
+To let the partition manager know which Kconfig ensures existence of a driver, the ``DEFAULT_DRIVER_KCONFIG`` option is used.
+Within :file:`partition_manager.cmake` the options is assigned :kconfig:option:`CONFIG_SOC_FLASH_NRF` for partitions on internal flash, which means that all partitions placed on internal flash area always available at run-time.
```

Comment:
```suggestion
Within the :file:`partition_manager.cmake` file the option is assigned the :kconfig:option:`CONFIG_SOC_FLASH_NRF` option for partitions in internal flash, which means that all partitions placed in internal flash are always available at run-time.
```

Reply:
There are other locations in your partition_manager.rst that also need to add " file" after any :file: reference, and " option" after any :kconfig:option: reference, as I did in the suggestion.